### PR TITLE
Update jQuery and Bootstrap

### PR DESCRIPTION
**Pull request by mauzey1**
Updating jQuery to version 3.6.0 to patch vulnerabilities.  This PR also updates the Bootstrap library.  There were also several javascript files that were unused that I have removed.

### CODE REVIEW

File: _includes/scripts.html [hunk 1]
```diff
@@ -1,19 +1,7 @@
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
-<script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.3/jquery-ui.min.js"></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
```
Comment (durack1) [inline _includes/scripts.html:1]: The latest version of jquery is 3.7.0 - see https://jquery.com/download/, this version is what is located at https://github.com/PCMDI/assets/tree/main/jquery

**Comment by mauzey1** [comment]
@durack1 Could you add [jquery-3.7.0.min.js](https://code.jquery.com/jquery-3.7.0.min.js) to [PCMDI/assets](https://github.com/PCMDI/assets/tree/main/jquery)

The problem with the 'slim' version of jQuery ([jquery-3.7.0.slim.min.js](https://github.com/PCMDI/assets/blob/main/jquery/jquery-3.7.0.slim.min.js)) is that it doesn't have the `getJSON` function, which some of the scripts on the site use.

**Comment by durack1** [comment]
Sure that could be added, did you have access to that repo? Feel free to add that yourself, I won't get to it until next week

**Comment by mauzey1** [comment]
@durack1 I have updated all of the jQuery and Bootstrap links to point [PCMDI/assets](https://github.com/PCMDI/assets)

**Review by durack1 — APPROVED** [review]
Looks good

**Pull request merged**